articles/virtual-machines/workloads/sap/deployment-guide.md

```diff
@@ -986,9 +986,7 @@ To install the Azure Extension for SAP by using Azure CLI:
 1. Install using Azure CLI 2.0
 
    1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
-+
-NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI must be installed.
```

Comment:
Please don't use NOTE section. IMHO this reduces the readability. Changing the text as described above should be good enough.